source/documentation/deploying-an-app/using-circleci-for-continuous-deployment.html.md.erb

```diff
@@ -143,6 +146,8 @@ jobs:
           fi
 ```
 
+Note -  Update the aws ecr login command above, if you are using AWS version 2.0, using reference [here][login-to-repo]
```

Comment:
Note - Update the aws ecr login command above if you are using AWS version 2.0. Click [here][login-to-repo] for reference.